### PR TITLE
Merge Jariullah.is-a.dev

### PR DESCRIPTION
**Pull request by Mohdjariullah**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/jariullah.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Personal Portfolio",
+  "repo": "https://github.com/Mohdjariullah/portfolio1",
+  "owner": {
+    "username": "mohdjariullah",
+    "email": "mohdjariullah@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://mohdjariullah.github.io/portfolio1/"
```
Comment (Stef-00012) [inline domains/jariullah.json:9]: CNAME can not contain `https://` or `/`

File: domains/mohdjariullah.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Personal Portfolio",
+    "repo": "https://github.com/Mohdjariullah/portfolio1/",
+    "owner": {
+        "username": "mohdjariullah",
+        "email": "mohdjariullah@gmail.com",
+        "twitter": "@mohdjariullah"
+    },
+    "record": {
+        "CNAME": "mohdjariullah.is-a.dev"
```
Comment (Stef-00012) [inline domains/mohdjariullah.json:10]: you can't use the domain itself as CNAME

**Review by Stef-DP — DISMISSED** [review]

File: domains/jariullah.json (hunk 1 above)
Comment (Mohdjariullah) [reply domains/jariullah.json:9]: factory.is-a.dev suggested this template to me with https:// and /

Comment (Mohdjariullah) [reply domains/jariullah.json:9]: image
@Stef-00012

File: domains/jariullah.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Personal Portfolio",
+  "repo": "https://github.com/Mohdjariullah/portfolio1",
+  "owner": {
+    "username": "mohdjariullah",
+    "email": "mohdjariullah@gmail.com"
+  },
+  "record": {
+    "CNAME": "mohdjariullah.github.io"
```
Comment (DEV-DIBSTER) [inline domains/jariullah.json:9]: Little content is provided on this site.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/jariullah.json (hunk 3 above)
Comment (Mohdjariullah) [reply domains/jariullah.json:9]: huh? Man it is a portfolio what else do you expect

Comment (DEV-DIBSTER) [reply domains/jariullah.json:9]: It has your name and the site name. That's not a portfolio that's two HTML lines.

Comment (Mohdjariullah) [reply domains/jariullah.json:9]: Sir that's not my portfolio 
https://portfolio1-sigma-three.vercel.app/
This is my portfolio1

Comment (21Z) [reply domains/jariullah.json:9]: @DEV-DIBSTER i believe this is his portfolio https://github.com/mohdjariullah/portfolio1/

Comment (Mohdjariullah) [reply domains/jariullah.json:9]: > @DEV-DIBSTER i believe this is his portfolio https://github.com/mohdjariullah/portfolio1/

Yep

File: domains/mohdjariullah.json (hunk 2 above)
Comment (orxngc) [inline domains/mohdjariullah.json:10]: What Stef said, fix this.

Comment (Mohdjariullah) [reply domains/mohdjariullah.json:10]: Is jariullah.json correct?

**Comment by Mohdjariullah** [comment]
Now is everything correct sir?

**Comment by Mohdjariullah** [comment]
I see this sir

**Pull request merged**